src/manifest.ts

```diff
@@ -40,21 +41,23 @@ export const createManifest = async ({ functions, path }: { functions: FunctionR
 }
 
 const formatFunctionForManifest = ({
```

Comment:
Nit: I'm not sure what this function is really doing still, the only thing I see is that it resolves the path :/

Reply:
I guess yeah resolving the path and limiting what we output. We might not return everything from `FunctionResult`, but haven't checked.